clang/lib/CodeGen/ItaniumCXXABI.cpp

```diff
@@ -3565,8 +3565,22 @@ void ItaniumRTTIBuilder::BuildVTablePointer(const Type *Ty) {
   // Check if the alias exists. If it doesn't, then get or create the global.
   if (CGM.getItaniumVTableContext().isRelativeLayout())
     VTable = CGM.getModule().getNamedAlias(VTableName);
-  if (!VTable)
-    VTable = CGM.getModule().getOrInsertGlobal(VTableName, CGM.Int8PtrTy);
+  if (!VTable) {
+    auto GVTy = VTableName == ClassTypeInfo || VTableName == SIClassTypeInfo
+                    ? CGM.Int8Ty->getPointerTo(/*addrspace =*/1)
```

Comment:
I'm a bit concerned that we have `1` hardcoded here. There should be some enum or target info we could use to say "global address space" instead of `1`.